packages/shared/styles/components/organisms/SfHero.scss

```diff
@@ -8,6 +8,17 @@
   color: var(--c-text);
   background-repeat: no-repeat;
   background-size: cover;
+  @include background(
```

Comment:
it was changed -  https://github.com/vuestorefront/storefront-ui/pull/1849/files